src/agent/legacy_schedule.rs

```diff
@@ -28,8 +28,10 @@ lazy_static! {
 }
 
 /// Weekly market hours schedule
+/// TODO: Remove after the migration
```

Comment:
nitpick: "after the migration" is not very specific. someone coming back to this todo won't necessarily know if it is done. Be precise: "after all publishers have upgraded to support the new schedule format"

Reply:
Makes sense, thanks. Fixed.